iosApp/Survey/Sources/Presentation/Views/SkeletonTextView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SkeletonTextView.swift
+//  Survey
+//
+//  Created by Bliss on 4/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SkeletonUI
+import SwiftUI
+
+struct SkeletonTextView: View {
+
+    let width: CGFloat
+    var lines: Int = 1
+
+    var body: some View {
+
+        let calcLines = CGFloat(lines)
```

Comment:
it is better to write fully the variable name instead of using abbreviations 😁
```suggestion
        let linesInFloat = CGFloat(lines)
```

Reply:
Fixed in https://github.com/blyscuit/kmm-swiftUI-IC/pull/59/commits/990883954214dc410b91eae965fe397e896990d8